### ISSUE TITLE
Expands directories is not working with cwd option

### ISSUE THREAD

**Issue by zeakd** [sindresorhus/globby#87]
Hi. with different `cwd`, expands directories is not working.

```
└── a/
  ├── b
  └── c/
    ├── d
    └── e
```

``` js
// in directory 'a'

globby('c') // ['c/d', 'c/e']
globby('c', { cwd: process.cwd() }) // ['c/d', 'c/e']
globby('a', { cwd: path.resolve(process.cwd(), '..') }) //  []
globby('a/**/*', { cwd: path.resolve(process.cwd(), '..') }) // ['a/b', 'a/c/d', 'a/c/e']
```

thanks

**Comment by thiagodp** [comment]
Same happening here, version `8.0.1` on Windows

**Comment by pvdlg** [comment]
This is due to https://github.com/kevva/dir-glob/issues/7.
The fix has been done a while ago but it's not released on npm...

**Comment by calebboyd** [comment]
@kevva can we bother you for an `npm publish`?

**Comment by sindresorhus** [comment]
The `dir-glob` version is published. @pvdlg What needs to be done on our side?

**Comment by pvdlg** [comment]
Yes we have to forward the `cwd` option (passed to `globby`) to `dir-glob`.

**Comment by pvdlg** [comment]
@sindresorhus after looking in more detail we don't have to do anything other than making a new release of `globby`. The last released version is `8.0.2` which locked to `dir-glob` dependency to `2.0.0`. I don't why...

That has been reversed in commits following `8.0.2`, so if we make a new release of globby we should be good.

**Comment by sindresorhus** [comment]
> The last released version is 8.0.2 which locked to dir-glob dependency to 2.0.0. I don't why...

See https://github.com/sindresorhus/globby/pull/99#issuecomment-452647305

**Comment by sindresorhus** [comment]
I have done a new release.
